src/style.css

```
@@ -0,0 +1,68 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+body {
+  margin-top: 20px;
+}
+
+.searchline {
+  font-family: Avenir, Helvetica, sans-serif;
+  font-size: 16px;
+  font-weight: 900;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  margin-bottom: 20px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px;
+  background-position-x: 26px;
+  background-position-y: center;
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  box-sizing: border-box;
```

move it to `*` selector to make it the same for all elements
```suggestion
	  * {
	  	box-sizing: border-box;	
	  }
```